satpy/readers/ami_l1b.py

```diff
@@ -212,3 +262,32 @@ def _calibrate_ir(self, dataset_id, data):
             bt_data = c0 + c1 * t_eff + c2 * t_eff * t_eff
             data.data = bt_data
         return data
+
+    def _get_rad_corr_factors(self):
+        """Retrieve radiance correction factors from user-supplied dict."""
+        try:
+            slope = self.radiance_correction[self.band_name]['slo']
+            offset = self.radiance_correction[self.band_name]['off']
```

Comment:
This method is duplicated from the ahi reader. Please consider factorizing, eg by creating a `RadianceCorrector` mixin class.

Reply:
Done.